package.json

```diff
@@ -34,9 +33,6 @@
     "xstream": "*"
   },
   "devDependencies": {
-    "@cycle/run": "3.1.0",
-    "typescript": "^2.3.4",
-    "typings": "^2.1.1",
-    "xstream": "^10.8.0"
```

Comment:
I think so.

I was getting a warning from npm because `@cycle/run` was both among the dependencies and devDependencies. `xstream` gets also installed without the devDependency. All in all everything compiles, with no warning. (I'd love to be able to give a better justification than "it runs", but I can't...)